daq/varz_state_collector.py

```diff
@@ -0,0 +1,132 @@
+"""Collect Varz states"""
+
+from __future__ import absolute_import
```

Comment:
lots of warnings here about this not being covered by tests -- do you know what's going on there?  Is this tested anywhere?

Reply:
I think it is because there is no unit test that covers this file. The reason I didn't add unit test because I think an Http server needs to be set up... Not sure if it's worth doing it as the varz collector is already being tested in test_mud.sh...

Btw, the warnings can be hidden in this page: https://docs.codecov.io/docs/github-checks#hiding-annotations-in-the-files-view

Reply:
Updated test_mud.sh so that it runs varz_state_collector.py with coverage instead of python. Most of the lines in varz_state_collector.py are covered by test now.